src/data/feedLibrary.js

```diff
@@ -78,14 +83,23 @@ export async function editBlogPost(
 
 /**
  * togglePinPost() pin/unpin a blog post
+ * @param {string} ref feedDAC post ref
```

Comment:
nit: for `ref` comments, you shouldn't use the abbreviation to explain the abbreviation. Applies to all locations. Also what is the reference? Is it the feedDAC post ID?